src/DependencyInjection/CmfCoreExtension.php

```diff
@@ -315,19 +312,37 @@ public function setupFormTypes(ContainerBuilder $container, LoaderInterface $loa
      */
     private function loadPublishWorkflow($config, XmlFileLoader $loader, ContainerBuilder $container)
     {
+        if (false === $config['enabled']
+            || ('auto' === $config['enabled'] && !class_exists(TokenStorageInterface::class))
+        ) {
+            $loader->load('no-publish-workflow.xml');
+
+            return;
+        }
+
+        if (!class_exists(TokenStorageInterface::class)) {
```

Comment:
You should use `interface_exists` when asking for an interface

Reply:
but i think, this will end in an warning, when the interface isn't there. but we will see.

Reply:
d'oh, thats why.

i just pushed with a change to check if SecurityBundle is in the list of kernel.bundles. that is what we really care about i guess.